targets/jm-servo-dual-23-2.0/profile/servo.c

```diff
@@ -23,11 +24,22 @@ const servo_params_t servo_params1 = {
     .power_pin = PC_15
 };
 
+    
+
+const relay_params_t hi_power = {
+    .relay_variant = JD_RELAY_VARIANT_ELECTROMECHANICAL,
+    .max_switching_current = 20,
+    .pin_relay_drive = PA_0,
+    .pin_relay_feedback = NO_PIN,
+    .pin_relay_led = NO_PIN,
+    .drive_active_lo = true,
+    .led_active_lo = false,
+    .initial_state = true
+};
+
 void app_init_services() {
     // high power mode when we get there
-    // pin_setup_output(PA_0);
-    // pin_set(PA_0, 0);
-    // pin_setup_input(PA_3, 1);
     servo_init(&servo_params0);
     servo_init(&servo_params1);
+    relay_service_init(&hi_power);
```

Comment:
I don't think a relay service is a good choice here. You definitely can't replace this relay with a normal relay or vice versa. It will do for testing though I guess

Reply:
Yes - the relay service is just for testing purposes. Ultimately we want high power when a power provider is connected to the bus!